app/domain/authentication/authn_jwt/restriction_validators/fetch_mandatory_claims.rb

```diff
@@ -0,0 +1,73 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    module RestrictionValidators
+      # Fetch the mandatory claims from the JWT authenticator policy which enforce 
+      # definition of annotations keys on JWT hosts 
+      FetchMandatoryClaims = CommandClass.new(
+        dependencies: {
+          resource_class: ::Resource,
+          fetch_required_secrets: ::Conjur::FetchRequiredSecrets.new,
+          parse_mandatory_claims: ::Authentication::AuthnJwt::InputValidation::ParseMandatoryClaims.new,
+          logger: Rails.logger
+        },
+        inputs: %i[authentication_parameters]
+      ) do
+        def call
+          fetch_mandatory_claims
+        end
+
+        private
+
+        def fetch_mandatory_claims
```

Comment:
Create function called mandatory claims with memoization

Reply:
I change it please review